rpc/handlers_test.go

```diff
@@ -2032,18 +2033,20 @@ func TestEstimateMessageFee(t *testing.T) {
 			assert.Equal(t, utils.MAINNET, network)
 			assert.NotNil(t, sequencerAddress)
 			assert.Len(t, paidFeesOnL1, 1)
+			assert.Equal(t, latestHeader.GasPrice, gasPrice)
 
-			return []*felt.Felt{expectedGasConsumed}, []json.RawMessage{{}}, nil
+			actualFee := new(felt.Felt).Mul(expectedGasConsumed, gasPrice)
+			return []*felt.Felt{actualFee}, []json.RawMessage{{}}, nil
 		},
 	)
 
-	gasConsumed, err := handler.EstimateMessageFee(msg, rpc.BlockID{Latest: true})
+	estimateFee, err := handler.EstimateMessageFee(msg, rpc.BlockID{Latest: true})
 	require.Nil(t, err)
 	require.Equal(t, rpc.FeeEstimate{
 		GasConsumed: expectedGasConsumed,
 		GasPrice:    latestHeader.GasPrice,
 		OverallFee:  new(felt.Felt).Mul(expectedGasConsumed, latestHeader.GasPrice),
```

Comment:
Because mock use callback to compute return values through `.DoAndReturn(...)`, inside callback I added computation for `actualFee`:
```
	actualFee := new(felt.Felt).Mul(expectedGasConsumed, gasPrice)
```